maintainers/maintainer-list.nix

```diff
@@ -169,6 +169,12 @@
     githubId = 12578560;
     name = "Quinn Bohner";
   };
+  _9999years = {
```

Comment:
please amend the commit message to be `maintainers: add _9999years`

Reply:
Done.